packages/medusa/src/api/admin/orders/[id]/fulfillments/route.ts

```diff
@@ -13,7 +14,10 @@ export const POST = async (
 
   const variables = { id: req.params.id }
 
-  const input = req.validatedBody
+  const input = {
+    ...((req.validatedBody as AdminOrderCreateFulfillmentType) ?? {}),
```

Comment:
suggestion: if you add the type in the template arg, this cast wont be necessary